recipes/numina/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set name = "numina" %}
+{% set version = "0.16" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2ce685760054a00bc34cd4d1fbef2359f7b17981a490286a67911432e9c7679f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - numina = numina.user.cli:main
+    - numina-bpm = numina.array.bpm:main
+    - numina-imath = numina.tools.imath:main
+    - numina-wavecalib = numina.array.wavecalib.__main__:main
+    - numina-ximshow = numina.array.display.ximshow:main
+    - numina-ximplotxy = numina.array.display.ximplotxy:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - toolchain
+    - numpy 1.11.*
+  host:
+    - python
+    - setuptools
+    - six >=1.7
+    - numpy >=1.11
+    - astropy >=1.3
+    - scipy
+    - pyyaml
+    - matplotlib
+    - enum34;python_version <"3.4"
+    - python-dateutil
+  run:
+    - python
+    - setuptools
+    - six >=1.7
+    - numpy >=1.7
+    - astropy >=1.3
+    - scipy
+    - pyyaml
+    - matplotlib
+    - enum34;python_version <"3.4"
+    - python-dateutil
+
+test:
+  imports:
+    - numina
+    - numina.array
+    - numina.array.display
+    - numina.array.peaks
+    - numina.array.peaks.tests
+    - numina.array.tests
+    - numina.array.trace
+    - numina.array.trace.tests
+    - numina.array.wavecalib
+    - numina.array.wavecalib.tests
+    - numina.core
+    - numina.core.tests
+    - numina.dal
+    - numina.dal.tests
+    - numina.drps
+    - numina.drps.tests
+    - numina.drps.tests.configs
+    - numina.ext
+    - numina.ext.gtc
+    - numina.frame
+    - numina.frame.tests
+    - numina.modeling
+    - numina.modeling.tests
+    - numina.simulation
+    - numina.store
+    - numina.store.gtc
+    - numina.store.tests
+    - numina.tests
+    - numina.tools
+    - numina.types
+    - numina.types.tests
+    - numina.user
+    - numina.user.tests
+    - numina.util
+    - numina.util.tests
+  commands:
+    - numina --help
+    - numina-bpm --help
+    - numina-imath --help
+    - numina-wavecalib --help
+    - numina-ximshow --help
+    - numina-ximplotxy --help
+  requires:
+    - pytest
```

Comment:
I don't think you need `pytest` to run the `imports` tests, since you are just importing the `test` modules, but not actually running any tests. You could probably drop `*.tests` from the imports since they don't do anything as far as I can tell from a quick glance.